docs/developing/octez-client/transactions.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Sending transactions
```

Comment:
This section is a bit tricky as it's called `Sending transactions`, but we have way more than that here, so maybe `Blockchain interactions` or `Transaction interactions` or just `Transactions with Octez` WDYT @timothymcmackin ?

Reply:
Hmmm. I like "sending transactions" because it's a task instead of a concept and because it's simple and something that developers need to do to test contracts. Maybe "Interacting with contracts"?

Reply:
yep, "Interacting with contracts" sounds good

Reply:
Changed to "Interacting with contracts"